pkg/ocm/clusters.go

```diff
@@ -264,7 +264,7 @@ func (c *Client) GetClusterByID(clusterKey string, creator *aws.Creator) (*cmv1.
 }
 
 func (c *Client) GetClusterUsingSubscription(clusterKey string, creator *aws.Creator) (*amv1.Subscription, error) {
-	query := fmt.Sprintf("plan.id = 'MOA' AND (display_name  = '%s' OR "+
+	query := fmt.Sprintf("plan.id = 'MOA' AND (name = '%s' OR "+
```

Comment:
Can you check if we have name in the subscription? i think we cant remove this

Reply:
Removing change